### DISCUSSION TITLE
Improving command usage strings

### DISCUSSION THREAD

**Discussion by mdcfe** [EssentialsX/Essentials#3892] [Ideas and Feature Suggestions] [answered]
Command usage strings are currently ambiguous, especially when commands have multiple functions with overlapping syntaxes. While overlapping syntaxes can be addressed in a future revision, it would be beneficial to users to improve the usage strings in 2.x.

Related: #3891

**Comment by JRoy** [comment, marked as answer]
8b23c2c4cd140afc0ed697ec4f691a3d7295682e
